tcpclient/connection.go

```diff
@@ -14,18 +14,22 @@ type Connection struct {
 type connectionStatus int
```

Comment:
This type was previously internal, but given the groupofconnection type is getting and checking this type as part of its methods (it consumes the "public" GetConnectionStatus), and the group of connections entity is brought by "multitcpclient" (makes sense.. tcpclient is scoped for single connections), now the type should be also public.

Reply:
This is also advised by codacy ;)

Reply:
👍